misc/python/materialize/mzbuild.py

```diff
@@ -295,6 +295,23 @@ def run(self) -> None:
                     Tagging={"TagSet": [{"Key": "ephemeral", "Value": ephemeral_str}]},
                 )
 
+            polar_signals_api_token = os.getenv("POLAR_SIGNALS_API_TOKEN")
+            # if self.rd.stable and polar_signals_api_token is not None:
```

Comment:
What's the difference between the commented condition and the one that we're actually using now below (that is, what is `self.rd.stable` signifying)?

Reply:
polar signals charges us for these, so we only upload on tagged version branches

Reply:
So it seems that we want to merge with this condition:

```python
if self.rd.stable and polar_signals_api_token is not None # currently commented out
```

as opposed to this:

```python
if polar_signals_api_token is not None # currently active
```

being enabled, no?

Reply:
yeah, you were looking at the "DNM testing" commit that I had 😉